easybuild/easyconfigs/r/RSGISLib/RSGISLib-2.2.0-foss-2015a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'RSGISLib'
+version = '2.2.0'
+
+homepage = "http://www.rsgislib.org"
+description = """The Remote Sensing and GIS software library (RSGISLib) is a
+ collection of tools for processing remote sensing and GIS datasets. The tools
+ are accessed using Python bindings or an XML interface."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://bitbucket.org/petebunting/rsgislib/downloads/']
+
+builddependencies = [
+    ('CMake', '3.4.1'),
+]
+
+dependencies = [
+    ('Boost', '1.60.0'),
+    ('CGAL', '4.8.1'),
+    ('GDAL', '1.11.5'),
+    ('GEOS', '3.5.0'),
+    ('GSL', '1.16'),
+    ('HDF5', '1.8.16'),
+    ('KEALib', '1.4.4'),
+    ('muParser', '2.2.5'),
+    ('Xerces-C++', '3.1.3'),
+]
+
+# Finish constructing configopts in this manner
+configopts  = " -DBOOST_INCLUDE_DIR=$EBROOTBOOST/include -DBOOST_LIB_PATH=$EBROOTBOOST/lib "
+configopts += " -DCGAL_INCLUDE_DIR=$EBROOTCGAL/include -DCGAL_LIB_PATH=$EBROOTCGAL/lib "
+configopts += " -DFFTW_INCLUDE_DIR=$EBROOTFFTW/include -DFFTW_LIB_PATH=$EBROOTFFTW/lib "
+configopts += " -DGDAL_INCLUDE_DIR=$EBROOTGDAL/include -DGDAL_LIB_PATH=$EBROOTGDAL/lib "
+configopts += " -DGEOS_INCLUDE_DIR=$EBROOTGEOS/include -DGEOS_LIB_PATH=$EBROOTGEOS/lib "
+configopts += " -DGMP_INCLUDE_DIR=$EBROOTGMP/include -DGMP_LIB_PATH=$EBROOTGMP/lib "
+configopts += " -DGSL_INCLUDE_DIR=$EBROOTGSL/include -DGSL_LIB_PATH=$EBROOTGSL/lib "
+configopts += " -DHDF5_INCLUDE_DIR=$EBROOTHDF5/include -DHDF5_LIB_PATH=$EBROOTHDF5/lib "
+configopts += " -DKEA_INCLUDE_DIR=$EBROOTKEALIB/include -DKEA_LIB_PATH=$EBROOTKEALIB/lib "
+configopts += " -DMPFR_INCLUDE_DIR=$EBROOTMPFR/include -DMPFR_LIB_PATH=$EBROOTMPFR/lib "
```

Comment:
@valtandor GMP and MPFR are indirect deps? maybe add a comment in the `dependencies` list via which dep they're pulled in?